uint32 type
string value

# Type values
uint32 ANSWER=0
uint32 ANIMATION=1
uint32 PICTURE=2
